docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,158 @@
+
+We will retrieve 10 sequences from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__ and select **Login**.
+
+   .. image:: ../images/gisaid-homepage.png
+      :width: 400
+      :alt: GISAID login link
+
+2. Login to your GISAID account.
+
+   .. image:: ../images/gisaid-login.png
+      :width: 200
+      :alt: GISAID login
+
+3. In the top left navigation bar, select **EpiCoV** then **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+4. Select the first 10 sequences.
+
+   .. image:: ../images/gisaid-select-sequences.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+5. Select **Download** in the bottom right of the search results.
+6. Select **Input for the Augur pipeline** as the download format.
+
+   .. image:: ../images/gisaid-augur-pipeline-download.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You may see different download options, but it is fine as long as **Input for the Augur pipeline** is available.
+
+7. Select **Download**.
+8. Download/move the ``.tar`` file into the ``ncov/data/`` directory.
+9. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+10. Rename the files as ``custom.metadata.tsv`` and ``custom.sequences.fasta``.
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a pre-written ``--configfile``:
+
+.. code:: text
+
+   nextstrain build . --cores all --configfile ncov-tutorial/custom-data.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can investigate the contents of ``custom-data.yaml`` (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       sequences: https://data.nextstrain.org/files/ncov/open/reference/sequences.fasta.xz
+     - name: custom_data
+       metadata: data/custom.metadata.tsv
+       sequences: data/custom.sequences.fasta
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     custom-build:
+       title: "Build with custom data and example data"
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+This is the same as the previous file, with some additions:
+
+1. A second input for the custom data, referencing the metadata and sequences files downloaded from GISAID.
+2. A ``builds`` section that defines one output :term:`docs.nextstrain.org:dataset` using:
+
+   1. A custom name ``custom-build``
```

Comment:
I suggest a bit more verbosity here:

1. A custom name ``custom-build``, which will be used to create the dataset filename, in this case `auspice/ncov_custom-build.json`
2. A custom title ``Build with custom data and example data``, which will be shown when you visualise the dataset in Auspice.
3. An Auspice config file,  `ncov-tutorial/auspice-config-custom-data.json`, which defines various parameters for how the data should be visualised in Auspice. It has the following contents:

Reply:
Done: da49a0ae